pkg/keyctl/keyring.go

```diff
@@ -3,11 +3,14 @@
 // license that can be found in the LICENSE file.
 
 // +build linux
-// +build 386 amd64
 
 // Package keyctl is a Go interface to linux kernel keyrings (keyctl interface)
```

Comment:
Non-blocking, RFC: Maybe add a note here that the package is deprecated, and most callers should call `x/sys/unix` directly.